lti_consumer/plugin/compat.py

```diff
@@ -23,29 +25,72 @@ def run_xblock_handler_noauth(*args, **kwargs):
 
 def load_block_as_anonymous_user(location):
     """
-    Load a block as anonymous user.
+    Load a block as an user if given, else anonymous user.
```

Comment:
Does this change still makes sense? Unless I misunderstood the code, it always impersonates the AnonymousUser.

Reply:
@viadanna, you're right. Fixed the misleading docstring.